index.php

```diff
@@ -7,6 +7,35 @@
 // Creates a variable which points to the correct database and gives username and password
 $db = new PDO('mysql:host=db;dbname=dinosaurs;', 'root', 'password');
 
+/**
+ * Checks if the form has been submitted and if so returns the dinos from the submitted search. If not, returns all dinos
+ *
+ * @param PDO $db database
+ * @return string of dinos to be displayed
+ */
+function checkIfSearched(PDO $db): string {
+    if (isset($_GET['submit'])) {
+        return Museum::displayAllDinos(DinosaurHydrator::getSearchedDinos($db, $_GET['search'], $_GET['filter']));
```

Comment:
Still not checking if the Get's exist, although it may work without I still think it's correct to put if(isset($_GET['submit'] && isset($_GET['search'] && isset($_GET['filter'])

Reply:
Please check previously resolved comments you made rather than saying "still not" as I had spoken to Ash and replied to your comment :)

Reply:
If you made this comment previously and I resolved it please check the resolved first before commenting again (especially with "still not") as I gave info as to why I resolved it without changing - in this case I asked Ash and it wasn't necessary